### PR TITLE
header currently showing on index only

### PR DESCRIPTION
**Pull request by safiay**
Problem: Remove the main content from all pages apart from the index

Solution: Added a route that specifically states that the header should show in index alone. 

Jira issue - https://dfacademy.atlassian.net/jira/software/projects/AT/boards/4?selectedIssue=AT-101

### CODE REVIEW

File: src/components/NavBar.js [hunk 1]
```diff
@@ -46,6 +47,7 @@ const NavBar = () => {
 </div>
 
           <Switch>
+            <Route path ="/" render={ ( props ) => ( props.location.pathname === "/") && <Header /> }></Route>
```
Comment (dearshrewdwit) [inline src/components/NavBar.js:50]: What's the reason for the anon function that checks `props.location.pathname === "/"`

Could you use `exact` path?
```js
<Route exact path="/">
  <Header />
</Route>
```

Comment (safiay) [reply src/components/NavBar.js:50]: We just did and pushed it, now we are waiting on the jenkins tests

Comment (dearshrewdwit) [reply src/components/NavBar.js]: Do you need the render attribute at all?

would just this work?
```js
<Route exact path="/"> 
  <Header />
</Route>
```

Comment (safiay) [reply src/components/NavBar.js]: I've changed it and it still works. This change has been pushed.

Comment (dearshrewdwit) [reply src/components/NavBar.js]: Nice

**Review by dearshrewdwit — CHANGES_REQUESTED** [review]
Ok good. 

But check the result - the navbar is sticky, and the content below is hiding partially under it
Screenshot 2021-09-02 at 17 43 24

To fix - add padding or margin so that the content is pushed down the page as much as the navbar takes up

**Comment by safiay** [comment]
The styling has been fixed. Please review this.

**Comment by dearshrewdwit** [comment]
> The styling has been fixed. Please review this.

Ok - slight gap on my device. 
Screenshot 2021-09-02 at 18 13 25


But i'll approve...

**Review by dearshrewdwit — APPROVED** [review]

**Review by breeshidy — APPROVED** [review]

**Comment by safiay** [comment]
lol thanks :)

**Pull request merged**